app/ai-health-coach/2025/09/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: AI Health Coach discovery summary
+description: An overview of the research, analysis, and conclusions from discovery phase
+date: 2025-09-24
+tags:
+  - discovery
+  - AI
+---
+
+This discovery explored the challenges faced by users of prevention and behaviour change services. It also examined whether these challenges could be addressed through the application of Artificial Intelligence, with a particular focus on AI’s potential to provide health coaching.
+
+## Our problem statement
+
+> Users of prevention services often experience support that feels disconnected or incomplete, and struggle with tools that only help with one aspect of their health – rather than supporting their broader, interconnected needs. While AI has the potential to improve personalisation and connection across services, it’s currently unclear how best to use AI to support across all aspects of someone’s prevention journey.
+
+## Our users
+
+Our users are those participating in (or could benefit from) healthy behaviour change or prevention services. Our review of existing research focused on users at various stages of that journey:
+
+- Better Health users and NHS App users
+- adults in the UK aged between 18 and 60, who were mostly from deprived areas (socioeconomic groups C2, D, E)
+- people with at least one raised health risk factor (being overweight, being inactive, smoking, heavier drinking)
+
+Other teams across NHS England and Department for Health and Social Care had already carried out extensive research to understand the pain points and barriers across this journey, so we drew upon these rather than conducting new primary research.
+
+## Research and analysis
+
+We identified pain points and user needs, as well as assess how these are being met by the existing landscape. We explored:
+
+### The existing prevention and behaviour change journey
+
+We mapped the user journey based on what we knew about prevention and behaviour change services from our experience across NHS England and Better Health. The journey was split into 9 sections, and 4 high‑level stages:
+
+1. become aware and engage
+- awareness for change
+- seek help
+2. find and select a service
+- find a prevention service
+- select a service
+- wait for service to start
+3. enrol and engage in a service
+- first contact and triage
+- ongoing service engagement
+- completion of service engagement
+4. feedback and next steps
+- feedback and next steps
```

Comment:
I think the sub-lists can be indented to make them slightly visually clearer? Not sure if that works or not though in Markdown.

```suggestion
1. become aware and engage
   - awareness for change
   - seek help
2. find and select a service
   - find a prevention service
   - select a service
   - wait for service to start
3. enrol and engage in a service
   - first contact and triage
   - ongoing service engagement
   - completion of service engagement
4. feedback and next steps
   - feedback and next steps
```